src/common/modules/UnicodeFontHandler.js

```diff
@@ -5,146 +5,135 @@ import * as BrowserCommunication from "/common/modules/BrowserCommunication/Brow
  * @returns {string}
  */
-function toggleCase(atext) {
-	let output = '';
-
-	for (let letter of atext) {
-		const upper = letter.toLocaleUpperCase();
-		const lower = letter.toLocaleLowerCase();
-		if (letter === lower && letter !== upper)
-			letter = upper;
-		else if (letter === upper && letter !== lower)
-			letter = lower;
-		output += letter;
-	}
-
-	return output;
+function toggleCase(text) {
+    let output = "";
+
+    for (let letter of text) {
+        const upper = letter.toLocaleUpperCase();
+        const lower = letter.toLocaleLowerCase();
+        if (letter === lower && letter !== upper) {
+            letter = upper;
+        } else if (letter === upper && letter !== lower) {
+            letter = lower;
+        }
+        output += letter;
+    }
+
+    return output;
 }
 
 /**
  * Change Case
  *
- * @public
+ * @private
  * @const
  * @type {Object.<string, function>}
  */
 const changeCase = Object.freeze({
-	"lowercase": (str) => str.toLocaleLowerCase(),
-	"uppercase": (str) => str.toLocaleUpperCase(),
-	"capitalize-each-word": (str) => capitalizeEachWord(str.toLocaleLowerCase()),
-	"toggle-case": (str) => toggleCase(str)
+    [CASE.CAPITALIZE]: (text) => capitalizeEachWord(text.toLocaleLowerCase()),
+    [CASE.LOWERCASE]: (text) => text.toLocaleLowerCase(),
+    [CASE.TOGGLE]: (text) => toggleCase(text),
+    [CASE.UPPERCASE]: (text) => text.toLocaleUpperCase(),
 });
 
 /**
- * Handle context menu click.
+ * Handle context menu selection.
  *
  * @param {Object} info
  * @param {Object} tab
  * @returns {void}
  */
-function handle(info, tab) {
-	let text = info.selectionText;
-
-	if (text) {
-		text = text.normalize();
-		let output = '';
-
-		switch (info.menuItemId) {
-			case "lowercase":
-			case "uppercase":
-			case "capitalize-each-word":
-			case "toggle-case":
-				output = changeCase[info.menuItemId](text);
-				break;
-			case "superscript":
-			case "small-caps":
-			case "all-small-caps":
-			case "unicase":
-			case "serif-bold":
-			case "serif-italic":
-			case "serif-bold-italic":
-			case "sans-serif":
-			case "sans-serif-bold":
-			case "sans-serif-italic":
-			case "sans-serif-bold-italic":
-			case "script":
-			case "script-bold":
-			case "fraktur":
-			case "fraktur-bold":
-			case "monospace":
-			case "double-struck":
-			case "circled":
-			case "circled-(black)":
-			case "squared":
-			case "squared-(black)":
-			case "fullwidth": {
-				output = changeFont(text, info.menuItemId);
-				break;
-			}
-		}
-
-		browser.tabs.executeScript(tab.id, {
-			code: `insertIntoPage("${output}");`,
-			frameId: info.frameId
-		});
-	}
+function handleMenuSelect(info, tab) {
+    let text = info.selectionText;
+
+    // ignore, if no text is selected
+    if (!text) {
+        return;
+    }
+
+    text = text.normalize();
+    let output = "";
+
+    // convert string back to Symbol
+    const caseId = caseByString[info.menuItemId];
+    const fontId = fontByString[info.menuItemId];
+    if (caseId) {
```

Comment:
Maybe `if (info.menuItemId in caseByString) {` to eliminate the temporary variables. Same below.